src/crunker.js

```diff
@@ -47,8 +42,8 @@ class Crunker {
   concatAudio(buffers) {
     let output = this._context.createBuffer(
         1,
-        sampleRate * this._totalLength(buffers),
-        sampleRate
+        this._totalLength(buffers),
```

Comment:
I'm really not sure about this line. I'm pretty sure it's wrong, but I am not sure what the "correct" value would be to make the tests pass.

Inspection came from debugging the following tests:
`it("returns a single buffer when concatenating")`
`it("uses correct length when concatenating")`

What happens is maxDuration is a pretty large number, multiplying this with the sampleRate (44100) makes this go out of bounds, which produces is a `NotSupportedError`. You can read about this [here](https://www.w3.org/TR/webaudio/#dom-baseaudiocontext-createbuffer).

Reply:
This is correct, missed this error in previous PR. 

`_totalLength` was previously `_maxDuration`, the maxDuration had to be multiplied by the sampleRate but totalLength does now.